src/vs/workbench/contrib/debug/browser/repl.ts

```diff
@@ -302,9 +303,13 @@ export class Repl extends Panel implements IPrivateReplService, IHistoryNavigati
 	layout(dimension: dom.Dimension): void {
 		this.dimension = dimension;
 		if (this.tree) {
+			const lastElementVisible = this.tree.scrollTop + this.tree.renderHeight >= this.tree.scrollHeight;
 			const treeHeight = dimension.height - this.replInputHeight;
 			this.tree.getHTMLElement().style.height = `${treeHeight}px`;
 			this.tree.layout(treeHeight, dimension.width);
+			if (lastElementVisible) {
```

Comment:
I do not really like this call here.
When the repl gets layouted if the last element is visible we should reveal it?
Why should we reveal it if it is already visible?
Also on layout seems like a wrong place to call this. Layout happens every time the dimensions of the repl change and on the first render.

Reply:
I put that extra call to reveal the last element when the inputbox gets resized because otherwise it would be covered. Also when resizing the windows.
![console](https://user-images.githubusercontent.com/25115070/59193023-e6a5a680-8b49-11e9-8d94-f973e3cc75b1.gif)